scripts/meili_food_index_swap.py

```
@@ -0,0 +1,229 @@
+``MEILI_SWAP_CANDIDATE_INDEX`` or ``MEILI_SWAP_CANDIDATE_UID``, optional batch/timeouts.
+
+RU: Операторский CLI для zero-downtime смены индекса без публичного HTTP-роута.
+"""
+
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import os
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)